### PR TITLE
Note refactors

### PR DESCRIPTION
**Pull request by Benjaminpjacobs**
refactors based on casey's comments

### CODE REVIEW

File: lib/events.js [hunk 1]
```diff
@@ -83,46 +129,49 @@ $(document).ready(function() {
...
             calories: $newFoodCalories.val()
         }
-
-        if (Food.validate(food)) {
+        if (Food.validate(food, $('.input-food'), $('.input-calories')) === true) {
             Food.addNew(food, $foodIndex);
-            $newFoodName.val('');
-            $newFoodCalories.val('');
+            clearFields([$newFoodName, $newFoodCalories]);
             $('.notify').remove();
```
Comment (podoglyph) [inline lib/events.js:145]: excellent refactor. I knew this needed help!

**Review by podoglyph — APPROVED** [review]

File: lib/events.js [hunk 2]
```diff
@@ -56,11 +61,52 @@ $(document).ready(function() {
     }
 
     const resetIndex = () => {
-        $('input:checkbox:checked').prop('checked', false);
+        $('.add-food-check input:checkbox:checked').prop('checked', false);
```
Comment (case-eee) [inline lib/events.js:64]: 👍

File: lib/events.js [hunk 3]
```diff
@@ -83,46 +129,49 @@ $(document).ready(function() {
...
             name: $newFoodName.val(),
             calories: $newFoodCalories.val()
         }
-
-        if (Food.validate(food)) {
+        if (Food.validate(food, $('.input-food'), $('.input-calories')) === true) {
             Food.addNew(food, $foodIndex);
-            $newFoodName.val('');
-            $newFoodCalories.val('');
+            clearFields([$newFoodName, $newFoodCalories]);
```
Comment (case-eee) [inline lib/events.js:144]: 👍

**Review by case-eee — COMMENTED** [review]
@Benjaminpjacobs @podoglyph great refactorings! also - great work adding some tests here 👍

**Pull request merged**